ntp-proto/src/algorithm/standard/clock_controller.rs

```diff
@@ -368,6 +373,7 @@ mod tests {
         last_max_error: RefCell<Option<NtpDuration>>,
         last_poll_interval: RefCell<Option<PollInterval>>,
         last_leap_status: RefCell<Option<NtpLeapIndicator>>,
+        last_algorithm_status: RefCell<Option<bool>>,
```

Comment:
what does that bool mean? could it be a custom type?

Reply:
It represents whether the user of the test clock has enabled or disabled the ntp discipline. You could make this a custom type, but given that it is part of testing code, I dont see the point (it would just create more code to understand when a test fails).

Reply:
fine, maybe rename the field to `last_ntp_discipline_enabled` then? "status = true" is rather meaningless if you're just reading the code.

Reply:
done.